apps/video/js/view.js

```diff
@@ -188,6 +192,17 @@ navigator.mozSetMessageHandler('activity', function viewVideo(activity) {
     }
   }
 
+  function setVideoAccessibility(asButton) {
```

Comment:
Same issue as in video.js regarding whether videoControls element can be used to store the aria label.